server.go

```diff
@@ -59,27 +63,40 @@ func (gs *CasbinServerHandler) serve(w http.ResponseWriter, r *http.Request) {
 		glog.Errorf("Filed to load the policies: %v", err)
 		return
 	}
-	if e.HasPermissionForUser(user, operation_name) == true {
-		response := v1.AdmissionReview{
+	/*	if e.HasPermissionForUser(user, operation_name) == true {
+		aresponse := v1.AdmissionReview{
```

Comment:
aresponse?

Reply:
aresponse is just an abbreviation for [Admission Response](https://pkg.go.dev/k8s.io/api/admission/v1#AdmissionResponse) which is used k8s [api](https://pkg.go.dev/k8s.io/api@v0.21.2/admission/v1).

Reply:
What about just using `admissionResponse`? `aresponse` seems to be a typo to people like me.